src/lib.rs

```diff
@@ -41,6 +41,18 @@ const KERNEL_FILE_NAME: &str = "kernel-x86_64";
 const RAMDISK_FILE_NAME: &str = "ramdisk";
 const CONFIG_FILE_NAME: &str = "boot.json";
 
+#[cfg(all(feature = "embedded_binaries", feature = "uefi"))]
+static UEFI_BOOTLOADER: &'static [u8] = include_bytes!(env!("UEFI_BOOTLOADER_PATH"));
+
+#[cfg(all(feature = "embedded_binaries", feature = "bios"))]
+static BIOS_BOOT_SECTOR: &'static [u8] = include_bytes!(env!("BIOS_BOOT_SECTOR_PATH"));
+#[cfg(all(feature = "embedded_binaries", feature = "bios"))]
+static BIOS_STAGE_2: &'static [u8] = include_bytes!(env!("BIOS_STAGE_2_PATH"));
+#[cfg(all(feature = "embedded_binaries", feature = "bios"))]
+static BIOS_STAGE_3: &'static [u8] = include_bytes!(env!("BIOS_STAGE_3_PATH"));
+#[cfg(all(feature = "embedded_binaries", feature = "bios"))]
+static BIOS_STAGE_4: &'static [u8] = include_bytes!(env!("BIOS_STAGE_4_PATH"));
```

Comment:
Instead of statics, I think we can use consts here. E.g. `const BIOS_BOOT_SECTOR: &'static [u8] = ...`.